testui/support/appium_driver.py

```diff
@@ -437,13 +437,15 @@ def __local_run(url, desired_caps, use_port, udid, log_file):
                 out.close()
                 break
             out.close()
-        return (
-            f"http://localhost:{port.__str__()}/wd/hub",
-            desired_caps,
-            process,
-            file_path,
-        )
-    return url, desired_caps, None
+        # Check Appium Version
+        result = subprocess.run(["appium", "-v"], stdout=subprocess.PIPE).stdout
+        url = f"http://localhost:{port.__str__()}/wd/hub"
+        if result.decode('utf-8').startswith("2."):
+            # for Appium version > 2.0.0
+            url = f"http://localhost:{port.__str__()}"
+        return url, desired_caps, process, file_path
+
+    return url, desired_caps, None, None
```

Comment:
Is it okay to add the extra `None` parameter here, compared to the original code?

Reply:
And there are 2 None variables because one is the process running the appium instance, and the other is the file path to the logs that are being generated when creating the appium instance and during execution